go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/defenseunicorns/zarf
 
-go 1.21.8
```

Comment:
Are we bumping the Go version on purpose or is there a new dependency which requires Go 1.22?

Reply:
Yes flux 1.3.0 requires go 1.22